src/Native/Runtime/gcrhenv.cpp

```diff
@@ -1504,6 +1416,86 @@ bool GCToEEInterface::EagerFinalized(Object* obj)
     return false;
 }
 
+bool GCToEEInterface::IsGCThread()
+{
+    Thread* pCurrentThread = ThreadStore::RawGetCurrentThread();
+    return pCurrentThread->IsGCSpecial() || pCurrentThread == ThreadStore::GetSuspendingThread();
+}
+
+bool GCToEEInterface::WasCurrentThreadCreatedByGC()
+{
+    return ThreadStore::RawGetCurrentThread()->IsGCSpecial();
+}
+
+struct ThreadStubArguments
+{
+    void (*m_pRealStartRoutine)(void*);
+    void* m_pRealContext;
+    bool m_isSuspendable;
+    bool m_hasStarted;
```

Comment:
`m_hasStarted` is not necessary in CoreRT. The place where it is returned can just return `true`.